<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>789</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1318</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_ID">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>9</y>
      <width>18</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>9</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>50</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Region">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Регион</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Region">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Population">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>95</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Население</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Population">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Area">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Площадь</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Area">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="openDatabaseButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Открыть базу данных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addRecordButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>130</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteRecordButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>170</y>
      <width>145</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewDataButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>90</y>
      <width>242</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Просмотреть базу данных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="centralRegionButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>250</y>
      <width>196</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Центральный регион</string>
    </property>
   </widget>
   <widget class="QPushButton" name="privolzhskyRegionButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>210</y>
      <width>198</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Приволжский регион</string>
    </property>
   </widget>
   <widget class="QPushButton" name="populationASCButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>290</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сортировка по возрастанию населения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="areaDESCButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>330</y>
      <width>324</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сортировка по убыванию площади</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createDatabaseButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>196</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Создать базу данных</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>521</width>
      <height>741</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
